### DISCUSSION TITLE
Download on loom.com fails

### DISCUSSION THREAD

**Discussion by CirrusShield** [aclap-dev/video-downloadhelper#574] [Help and Q&A]
### Your question

Hello, I am trying to download videos from loom.com but the download fails.
Here are the bug details:
{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.skool.com/mentoring-revelation-2460/classroom/69f84fb0?md=be75a8a5615841a7ba13430f1366735d",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0000028fc0be7b40] Error when loading first segment 'https://luna.loom.com/id/0b324ccc4ecf464e829602b3f8c73250/rev/6c00147c7f34d5c075ba565adfc58ea70226c3cae8d0bafae1862b31d7477e0f5/resource/hls/0b324ccc4ecf464e829602b3f8c73250-video-0.ts'\r\n[in#0 @ 0000028fc0be79c0] Error opening input: Invalid data found when processing input\r\nError opening input file https://luna.loom.com/id/0b324ccc4ecf464e829602b3f8c73250/rev/6c00147c7f34d5c075ba565adfc58ea70226c3cae8d0bafae1862b31d7477e0f5/resource/hls/mediaplaylist-video-bitrate3200.m3u8?Signature=hPDqibNqHwleN4DqimPNoxY5VW7RAyUUGAtHuwOBRfJun37rbubLkb8tV3phV4qY-vdbycaH0OM5YjnRNpHJ9~Jx6l3gA~iXv78GAuqhEsGA-Q9RarI4PXw5kxUOQqek5BkxHRfFSALu-~RMWiEnQeLq~kusNhbR9buvMZ6uVAqzBzXi9~XbPAc-oYD9yVfbdSHV1MSDeSkazqZ03TeM6GlVjdOnwWDcARYq9K72JC~TaLBEe8vP7cqQnC8GoJC6tdCJZltEkRKK~Tc57lzDV2KI8mVzzLK8ro-J3fpi5Ea3eUEoa2R-H-1RdpidWJycfzmeonLxHOQHZTw7mu~mCQ__&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9sdW5hLmxvb20uY29tL2lkLzBiMzI0Y2NjNGVjZjQ2NGU4Mjk2MDJiM2Y4YzczMjUwL3Jldi82YzAwMTQ3YzdmMzRkNWMwNzViYTU2NWFkZmM1OGVhNzAyMjZjM2NhZThkMGJhZmFlMTg2MmIzMWQ3NDc3ZTBmNS9yZXNvdXJjZS8qIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoxNzE2NTYyNzM3fX19XX0_&Key-Pair-Id=APKAJQIC5BGSW7XXK7FQ.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_c961c2c4-d92b-4b0f-a513-da33725d0810",
    "downloadable_id": "downloadable_4271185132898218"
  }
}

Please let me know if any additional test or information is needed.
Thanks

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"0e6c9ad9064d4d2a"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "D:\\My Documents\\Aliston\\OneDrive\\Aliston\\Aliston Team - Documents\\General\\Training Core Curriculum and KB\\Training Core Curriculum\\Excel Quick Tips\\6.Working with Files",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 11
}

**Comment by paulrouget** [comment]
Duplicate of https://github.com/aclap-dev/video-downloadhelper/discussions/7 - known issue I hope to fix in v9.1.

**Reply by mosesjaxon** [reply]
> Duplicate of #7 - known issue I hope to fix in v9.1.

@paulrouget , was this update included in 9.1? (also asked on #7 )

**Reply by Mattmjs** [reply]
It is not yet fixed regarding loom. Wait it out for that fix or use yt-dlp.
